js/functions.js

```diff
@@ -0,0 +1,31 @@
+//Функция для проверки длины строки
+
+const checkStringLength = function (string, stringLength) {
+  if (string.length > stringLength){
+    return false;
+  }
+  return true;
```

Comment:
Выражение можно упростить
```suggestion
  return string.length <= stringLength;
```